app/views/accounttransfers/view_accounttransfer.html

```diff
@@ -14,7 +14,7 @@ <h4>{{ 'label.heading.viewaccounttransferdetails' | translate}}</h4>
                     <table width="100%">
                         <tr>
                             <td width="20%">{{ 'label.heading.transactionamount' | translate }}</td>
```

Comment:
this seems like a very hard way to maintain the support of 6 decimal places. Shouldn't it at least be stored in a config variable or something?

Reply:
@nassan : constant has been added for decimal points in configurations.js file and using the same in UI.